src/reproschema_to_fhir/config.py

```diff
@@ -25,4 +26,19 @@ def get_language(self):
         return self.LANGUAGE
     
     def get_mode(self):
-        return self.MODE
\ No newline at end of file
+        return self.MODE
+
+    def set_questionnaire(self, questionnaire):
```

Comment:
getters and setters aren't really pythonic (all attributes are public). you should just set the attribute normally, e.g. instead of `object.set_questionnaire(value)`, just do `object.questionnaire = value`. same for all the other cases.

(now, *if* you were doing something else every time the attribute was set, like say you wanted to update another attribute if the questionnaire attribute is changed, then you could use a getter/setter, but I don't see that here! and if you want to do this, then you should check out the property decorator)